### PR TITLE
www-apps/redmine: security updates for all versions (#634602)

### PR DESCRIPTION
**Pull request by winterheart**
Update to 3.2.8, 3.3.5, 3.4.3, fixes multiple XSS vulnerabilities.

https://bugs.gentoo.org/show_bug.cgi?id=634602

### CODE REVIEW

File: www-apps/redmine/redmine-3.4.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+	insinto /etc/logrotate.d
+	newins "${FILESDIR}"/redmine.logrotate redmine
+
+	newconfd "${FILESDIR}/${PN}.confd" ${PN}
+	newinitd "${FILESDIR}/${PN}-3.initd" ${PN}
+	doenvd "${T}/50${PN}"
+}
+
+pkg_postinst() {
+	einfo
```
Comment (jonasstein) [inline www-apps/redmine/redmine-3.4.3.ebuild]: the empty einfo lines are not needed here

File: www-apps/redmine/redmine-3.2.8.ebuild [hunk 2]
```diff
@@ -0,0 +1,215 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (jonasstein) [inline www-apps/redmine/redmine-3.2.8.ebuild]: You can bump to EAPI=6 so that all ebuilds have the same EAPI this makes maintenance easier

**Comment by monsieurp** [comment]
@winterheart ping?

**Comment by winterheart** [comment]
@monsieurp ready

**Pull request closed without merging**